directoryscr.js

```diff
@@ -0,0 +1,29 @@
+console.log("yo");
+
+function dostuff(membs, target){
```

Comment:
I'd like to see "dostuff" be refactored to something like addMembers or populateList or, heck, populateListOfMembers. I'm looking at cleaning up the html here per [the ul issue](https://github.com/ObieSource/obiesource.github.io/pull/2#discussion_r830632761) and will probably fix this while I'm at it.